### PR TITLE
dev-python/traits: version bump, cleanup ebuild

### PR DESCRIPTION
**Pull request by AndrewAmmerlaan**
Package-Manager: Portage-3.0.12, Repoman-3.0.2
Signed-off-by: Andrew Ammerlaan <andrewammerlaan@riseup.net>

### CODE REVIEW

File: dev-python/traits/traits-6.1.1.ebuild [hunk 1]
```diff
@@ -15,28 +15,20 @@ SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
 
 LICENSE="BSD"
 SLOT="0"
-KEYWORDS="~amd64 ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux"
-IUSE="test"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
```
Comment (epsilon-0) [inline dev-python/traits/traits-6.1.1.ebuild:18]: This package is maintained by @TheChymera and has been updated quite recently. 
I am not sure I want to update this without his input, also given that this is dropping tests. (?)

Comment (AndrewAmmerlaan) [reply dev-python/traits/traits-6.1.1.ebuild:18]: > Still, this one can be done by @TheChymera

Sure, that's alright.

At the moment I'm just going through the repoman output from top to bottom. This one only had a minor warning if I remember correctly, but I fixed some other things while I was at it.

**Comment by TheChymera** [comment]
@AndrewAmmerlaan I'm fine with this update and happy to merge once I test it locally. Do you have push access? Also, feel free to propose bumps to my packages in the future, just that in the interest of covering a broader set of packages it's better to prioritize those which don't have an active maintainer. A good way to check is to see who the maintainer is, and then query `git log` for whether they contributed anything in the last few months.

Also, pkgcheck is pretty lame and complains about things which are basically cosmetic. We should try to prioritize things that actually don't work. But I'm not sure how exactly.... I've also spent a lot of time recently fixing basically trivial pkgcheck complaints in the order I saw them :-/

**Comment by AndrewAmmerlaan** [comment]
> Do you have push access?

I do not have push access :cry: 

> Also, pkgcheck is pretty lame and complains about things which are basically cosmetic. We should try to prioritize things that actually don't work. 

Yeah I know, this is why I use `repoman dx full` instead of pkgcheck, it is less picky and tends to only show important things. At the moment I'm only fixing the packages which have missing dependencies.

**Comment by TheChymera** [comment]
@AndrewAmmerlaan closed in https://gitweb.gentoo.org/proj/sci.git/commit/?id=022d5da8a7b0fa7d16141aa2f250b10f7962adea

**Comment by TheChymera** [comment]
>  do not have push access :cry: 

You could ask for push access by opening a bug on bugzilla. You seem to have a lot of interest and a good grip on ebuilds. In the long run it makes little sense to wait on others to fix things. You may need to complete the devquiz to demonstrate your grip on ebuilding and the general Gentoo infrastructure (not that complicated, if you do it full-time it's done in a weekend for sure).

**Comment by AndrewAmmerlaan** [comment]
> > do not have push access cry
> 
> You could ask for push access by opening a bug on bugzilla. You seem to have a lot of interest and a good grip on ebuilds. In the long run it makes little sense to wait on others to fix things. You may need to complete the devquiz to demonstrate your grip on ebuilding and the general Gentoo infrastructure (not that complicated, if you do it full-time it's done in a weekend for sure).

Thank You, I'll open an access-request bug later today or tomorrow :smile:

**Comment by AndrewAmmerlaan** [comment]
> > > do not have push access cry
> > 
> > 
> > You could ask for push access by opening a bug on bugzilla. You seem to have a lot of interest and a good grip on ebuilds. In the long run it makes little sense to wait on others to fix things. You may need to complete the devquiz to demonstrate your grip on ebuilding and the general Gentoo infrastructure (not that complicated, if you do it full-time it's done in a weekend for sure).
> 
> Thank You, I'll open an access-request bug later today or tomorrow smile

Bug opened: https://bugs.gentoo.org/763534 :smile:

**Pull request closed without merging**